x-pack/plugins/reporting/public/management/components/report_info_flyout_content.tsx

```diff
@@ -186,19 +190,31 @@ export const ReportInfoFlyoutContent: FunctionComponent<Props> = ({ info }) => {
   ];
 
   const warnings = info.getWarnings();
-  const errored = info.getError();
+  const errored =
+    /*
+     * We link the user to documentation if they hit this error case. Note: this
+     * should only occur on cloud.
+     */
+    info.error_code === VisualReportingSoftDisabledError.code
+      ? sharedI18nTexts.cloud.insufficientMemoryError(
```

Comment:
Any reason to not make this message part of the `Job.getError` implementation?

Reply:
That's a good point... But I'd prefer to not raise this logic to a more general level yet. Because the i18n contains JSX we would no longer guarantee that `Job.getError` will only return a `undefined | string`.

I prefer having this code implemented in a more "dumb" way because it remains a bit easier to delete (something may still want to do one day).